babel.config.js

```diff
@@ -1,31 +1,38 @@
-module.exports = {
-    presets: [
+module.exports =
```

Comment:
@IgorChaikin Could you disable `eslint` rules which are conflicted with `prettier` formatting

![conflict-1](https://user-images.githubusercontent.com/12561302/126611569-04e82b59-9b12-48e6-968c-90b3552aeac6.gif)